webapp/templates/redirect-create.xhtml

```diff
@@ -0,0 +1,95 @@
+<html version="XHTML+RDFa 1.0" xmlns="http://www.w3.org/1999/xhtml"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+    xmlns:dcterms="http://purl.org/dc/terms/"
+    xmlns:calli="http://callimachusproject.org/rdf/2009/framework#">
+<head>
+    <title>New Redirect</title>
+    <link rel="help" href="../../callimachus-reference#PURL" target="_blank" title="Help" />
+    <script type="text/javascript" src="../scripts/purl.js"></script>
+</head>
+<body>
+    <form role="form" id="form" method="POST" action="" enctype="text/turtle" typeof="calli:Redirect" class="container">
+        <div class="page-header">
+            <h1>New Redirect</h1>
+        </div>
+        <div id="ruleOptions" style="display:none">
+            <!-- These are not used by the template but are required for validation when the resource is created -->
+            <input type="hidden" property="calli:canonical" content="?canonical" value="{?canonical}"/>
+            <input type="hidden" property="calli:alternate" content="?alternate" value="{?alternate}"/>
+        </div>
+        <div class="row">
+            <div class="col-sm-8">
+                <div class="row">
+                    <fieldset>
+                        <div class="form-group col-sm-12">
+                            <label for="local">Local name</label>
+                            <div>
+                                <input type="text" id="local" value="{rdfs:label}" class="form-control" required="required" autofocus="autofocus"
+                                    onchange="calli.updateProperty(event, 'rdfs:label')" />
+                            </div>
+                        </div>
+                        <div class="form-group col-sm-12">
+                            <label for="comment">Comment</label>
+                            <div>
+                                <textarea id="comment" class="form-control"
+                                    onchange="calli.updateProperty(event, 'rdfs:comment')">{rdfs:comment}</textarea>
+                            </div>
+                        </div>
+                    </fieldset>
+                </div>
+                <div class="row">
+                    <fieldset id="rules" class="rule">
+                        <div class="form-group col-sm-3">
+                            <label for="type">Request methods</label>
```

Comment:
I think this can be confusing (for target users). Just assume the redirect applies to all supported methods.